### ISSUE TITLE
(Template) TypeError: Cannot read properties of undefined

### ISSUE THREAD

**Issue by JeffreyArt1** [rx-angular/rx-angular#1579]
# Description
Apparently the new v16 update broke something.

This is the message I'm getting.

```
TypeError: Cannot read properties of undefined (reading 'config')
    at new RxLet (vendor.js:100016:210)
    at NodeInjectorFactory.RxLet_Factory [as factory] (vendor.js:100142:12)
    at getNodeInjectable (vendor.js:37828:38)
    at instantiateAllDirectives (vendor.js:44562:23)
    at createDirectivesInstances (vendor.js:43998:3)
    at Module.ɵɵtemplate (vendor.js:47933:5)
    at AppComponent_Template (main.js:193:65)
    at executeTemplate (vendor.js:43959:9)
    at renderView (vendor.js:45103:7)
    at renderComponent (vendor.js:45050:3)
```
# Steps to Reproduce the Issue

[Repro (Stackblitz)](https://stackblitz.com/edit/angular-xigbvh?file=src%2Fmain.ts)

# Environment
```
Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1601.0
@angular-devkit/build-angular   16.1.0
@angular-devkit/core            16.1.0
@angular-devkit/schematics      16.1.0
@angular/cli                    16.1.0
@nguniversal/builders           16.1.0
@nguniversal/express-engine     16.1.0
@schematics/angular             16.1.0
rxjs                            7.8.1
typescript                      4.9.5
```

Node versions: `v18.15.0` and `v20.2.0`

**Comment by daniel-zero** [comment]
I have the same error after upgrading to the latest rx-angular v16 release

**Comment by juri33** [comment]
same here after update

**Comment by CharlieGreenman** [comment]
same here 

cc @edbzn

**Comment by edbzn** [comment]
Fixed by #1581, thanks for raising the issue.
